src/main/java/org/fundacionjala/at15/pokemon/Trainer.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.at15.pokemon;
+
+public class Trainer {
+    private final int limit = 6;
+    private Wallet wallet = new Wallet();
+    private String[] teamPokemon = new String[limit];
```

Comment:
Once we have PokemonTeam class you should replace String[] -> PokemonTeam

Reply:
yes, it will be like that